core_backend/app/question_answer/config.py

```diff
@@ -2,6 +2,7 @@
 
 # Functionality variables
 N_TOP_SIMILAR = os.environ.get("N_TOP_SIMILAR", "4")
```

Comment:
Thought on changing this to `N_TOP_CONTENT_FOR_SEARCH` to be consistent? Ok if we just do it later.

Reply:
done!